Resources/Prototypes/Datasets/Names/diona.yml

```
@@ -37,186 +37,186 @@
   - Звезда
   - Путь
   - Облако
-  - Arctic
-  - Sea
-  - Lake
-  - Spring
-  - Swamp
-  - River
-  - Stream
-  - Forest
-  - Rainforest
-  - Soil
-  - Valley
-  - Moor
-  - Steppe
-  - Orchard
-  - Orchid
-  - Glacier
-  - Desert
-  - Tundra
-  - Taiga
-  - Plain
-  - Plains
-  - Plateau
-  - Cliff
-  - Coast
-  - Shore
-  - Jungle
-  - Tropics
-  - Flower
-  - Grass
-  - Bark
-  - Autumn
-  - Summer
-  - Winter
-  - Fruit
-  - Leaves
-  - Overgrowth
-  - Atmosphere
-  - Climate
-  - Chill
-  - Winds
-  - Blossom
-  - Bloom
-  - Cap
-  - Saturation
-  - Permeation
-  - Light
-  - Taste
-  - Acorn
-  - Shell
-  - Ice
-  - Monsoon
-  - Overcast
-  - Storm
+  - Арктика
+  - Море
+  - Озеро
+  - Весна
+  - Болото
+  - Река
+  - Поток
+  - Лес
+  - Дождевой Лес
+  - Почва
+  - Долина
+  - Топь
+  - Степь
+  - Фруктовый Сад
+  - Орхидея
+  - Ледник
+  - Пустыня
+  - Тундра
+  - Тайга
+  - Равнина
+  - Равнины
+  - Плато
+  - Скала
+  - Побережье
+  - Берег
+  - Джунгли
+  - Тропики
+  - Цветы
+  - Трава
+  - Кора
+  - Осень
+  - Лето
+  - Зима
+  - Фрукт
+  - Листья
+  - Заросли
+  - Атмосфера
+  - Климат
+  - Холод
+  - Ветра
+  - Цветение
+  - Расцвет
+  - Шапка
```

```suggestion
  - Шляпка
```
Шляпка гриба, например